main.go

```diff
@@ -65,5 +65,5 @@ func main() {
 		console.Fatalln("Go runtime version check failed.", err)
 	}
 
-	minio.Main(os.Args, os.Exit)
+	minio.Main(os.Args)
```

Comment:
I know passing os.Exit looks odd here but we do that to be able to test it. Running os.Exit(0) while testing will stop all tests and return success status.